browsers/file-io.mdx

```diff
@@ -30,103 +34,104 @@ const kernel = new Kernel();
 
 // Poll listFiles until the expected file appears in the directory
 async function waitForFile(
-  sessionId: string,
-  dir: string,
-  filename: string,
-  timeoutMs = 30_000
+sessionId: string,
```

Comment:
Hmm something looks odd with the formatting. Looks like it was all left aligned. Can you adjust?